segments/weather.sh

```diff
@@ -28,9 +23,9 @@ export TMUX_POWERLINE_SEG_WEATHER_GREP="${TMUX_POWERLINE_SEG_WEATHER_GREP_DEFAUL
 # 1. Go to Yahoo weather http://weather.yahoo.com/
 # 2. Find the weather for you location
 # 3. Copy the last numbers in that URL. e.g. "http://weather.yahoo.com/united-states/california/newport-beach-12796587/" has the numbers "12796587"
-export TMUX_POWERLINE_SEG_WEATHER_LOCATION=""
+export TMUX_POWERLINE_SEG_WEATHER_LOCATION="${TMUX_POWERLINE_SEG_WEATHER_LOCATION_DEFAULT}"
```

Comment:
This default value is not defined among the other default values, starting at line 6 above.

What would a good default value for a location be? I don't know, that's why there is not default set.

The intended usage here is simply that the user sets the value of `TMUX_POWERLINE_SEG_WEATHER_LOCATION` in `/.tmux-powerlinerc` to something.

:-)